xDrip/Scenes/Settings/BaseSettings/BaseSettingsViewController.swift

```diff
@@ -105,6 +120,22 @@ extension BaseSettingsViewController: UITableViewDelegate, UITableViewDataSource
         case .normal(let cells, _, _):
             handleSelectionForNormalCell(cells[indexPath.row])
             
+            switch cells[indexPath.row] {
+            case .pickerExpandable(_, _, _, _):
```

Comment:
You may omit associated values by just writing `case .pickerExpandable:` in such case.